### ISSUE TITLE
row chart title label needs dy offset

### ISSUE THREAD

**Issue by wojdyr** [dc-js/dc.js#941]
rowChart with renderTitleLabel(true) has vertically misaligned title labels:
![rowchart](https://cloud.githubusercontent.com/assets/491873/7960406/49e6e65a-09f8-11e5-85cf-bc861d90828e.png)

Both labels and titles have the same `y`, but labels have also `dy`.
Adding `dy` to titles fixes the problem.
```diff
diff --git a/src/row-chart.js b/src/row-chart.js
index d7a16de..347ae7a 100644
--- a/src/row-chart.js
+++ b/src/row-chart.js
@@ -241,6 +241,7 @@ dc.rowChart = function (parent, chartGroup) {
             var titlelab = rows.select('.' + _titleRowCssClass)
                     .attr('x', _chart.effectiveWidth() - _titleLabelOffsetX)
                     .attr('y', _labelOffsetY)
+                    .attr('dy', _dyOffset)
                     .attr('text-anchor', 'end')
                     .on('click', onClick)
                     .attr('class', function (d, i) {
```

**Comment by gordonwoodhull** [comment]
This looks like a reasonable fix. The `_dyOffset` fix for titles is relatively new and it just needs to be applied to labels as well.

**Comment by gordonwoodhull** [comment]
Thanks @wojdyr!

**Comment by adrm** [comment]
@gordonwoodhull Is this patch to be applied soon? I can submit the pull request with @wojdyr 's patch if he is not able to do it now.

**Comment by gordonwoodhull** [comment]
Yes, please do. There is a lot to keep track of in this project, and all pull requests help, even small ones. Thanks!

**Comment by gordonwoodhull** [comment]
Fixed in 2.0 beta 28. Thanks @adrm!
